save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/TestExecutionController.kt

```diff
@@ -54,34 +54,30 @@ class TestExecutionController(private val testExecutionService: TestExecutionSer
      * @param executionId an ID of Execution to group TestExecutions
      * @param page a zero-based index of page of data
      * @param size size of page
-     * @param status
-     * @param testFileName
-     * @param testSuiteName
-     * @param tag
+     * @param filters
      * @param authentication
      * @param checkDebugInfo if true, response will contain information about whether debug info data is available for this test execution
      * @return a list of [TestExecutionDto]s
      */
-    @GetMapping(path = ["/api/$v1/testExecutions"])
+    @PostMapping("/api/$v1/filteredTestExecutions")
```

Comment:
```suggestion
    @PostMapping("/api/$v1/test-executions")
```
We try to avoid these Java-style names in API endpoints. Also, why mention that it's filtered, if `filters` is an optional parameter?

Reply:
there is another test with the same name, so it was necessary to rename this test

Reply:
The second one, which returns a single `TestExecutionDto`, can be renamed actually, to `/test-execution` or even `/test-execution/by-location`

Reply:
I renamed my request to "test-execution"

Reply:
I renamed first request to `testExecutions` and second request to `testExecution`

Reply:
Did you rename call sites too?